package/packaging.md

```diff
@@ -1,9 +1,9 @@
-# Git-Town Packaging and Release Process
+# Git-Town Packaging and Release Process For Linux
 
-Instructions on how to update all the necessary packages once a new release is out
+This document contains instructions on how to update all the necessary packages once a new release is out.
 
-## Linux
 The current linux (all distros) maintainer is [allonsy](https://github.com/allonsy). If an update occurs, please bug him
+
 ### Arch Linux
 The package will need to be flagged out of date [here](https://aur.archlinux.org/packages/git-town/)
 This will tell [allonsy](https://github.com/allonsy) to update the package (most likely minimal intervention necessary). There is a button on the right hand side menu to flag the package.
```

Comment:
While you're updating this, can you please add periods to these sentences? They end up grouped together. Same for sentences under `Debian Based Distributions`